<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddJointLoadsDialog</class>
 <widget class="QDialog" name="AddJointLoadsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>231</width>
    <height>167</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Joint Loads</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="selectedJointsLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Selected Joints</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="2">
    <widget class="QTextEdit" name="selectedJointsTextEdit">
     <property name="undoRedoEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="acceptRichText">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1" rowspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="2">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="loadJointLabel">
       <property name="text">
        <string>Load Joint</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="ComboBox" name="loadJointComboBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="horizontalComponentLabel">
       <property name="text">
        <string>H (kN)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="horizontalComponentDoubleSpinBox">
       <property name="decimals">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="verticalComponentLabel">
       <property name="text">
        <string>V (kN)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="verticalComponentDoubleSpinBox">
       <property name="decimals">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="positionLabel">
       <property name="text">
        <string>Position</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="ComboBox" name="positionComboBox"/>
     </item>
    </layout>
   </item>
   <item row="2" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>42</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="addJointLoadsPushButton">
     <property name="text">
      <string>Apply</string>
     </property>
     <property name="icon">
      <iconset resource="../../rsc.qrc">
       <normaloff>:/icons/dialog-ok-apply.png</normaloff>:/icons/dialog-ok-apply.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>combobox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../rsc.qrc"/>
 </resources>
 <connections/>
</ui>
